projects/bunicorn/index.js

```diff
@@ -27,6 +29,58 @@ query GET_TOTAL_STABLE_TVL($block: Int) {
 }
 `;
 
+const graphBuniPriceQuery = gql`
```

Comment:
Why obtain buni price from the subgraph instead of just returning the amount of buni and letting us handle it through coingecko?

Reply:
I fixed it